esp/smc/SMCLib/TpCommon.cpp

```diff
@@ -256,6 +256,16 @@ extern TPWRAPPER_API void validateDropZoneAccess(IEspContext& context, const cha
         if (!isHostInPlane(dropZone, hostReq, true))
             throw makeStringExceptionV(ECLWATCH_INVALID_INPUT, "Host %s is not valid DropZone plane %s", hostReq, targetDZNameOrHost);
     }
+
+    //If the dropZonePath is an absolute path, change it to a relative path.
+    if (isAbsolutePath(fileNameWithRelPath))
```

Comment:
can we also add some comment to CWsFileIOEx::onReadFileData in this PR (to reduce future confusion) to note that despite the "destRelativePath" saying it's relative for legacy reason, it also supports absolute paths?

(I'm also curious why ReadFileDataRequest uses fields with "Dest" in them, when this is only used for reading ..)

Reply:
Will add the comment.
All of the three methods in WsFileIO uses the "Dest". But, it is not 'good' for the ReadFileDataRequest.